Gemfile.lock

```diff
@@ -277,18 +323,17 @@ GEM
     websocket-driver (0.7.5)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
+    xpath (3.2.0)
+      nokogiri (~> 1.8)
     zeitwerk (2.5.4)
 
 PLATFORMS
-  aarch64-linux-musl
```

Comment:
Can we add these platforms back?

Reply:
Sure. I didn't realise it was removed as I didn't explicitly remove it.
Not to worry I've added it back again in my PR.